crates/common/tedge_config_macros/Cargo.toml

```diff
@@ -15,7 +15,7 @@ once_cell = "1.17"
 serde = "1.0"
 tedge_config_macros-macro = { path = "macro" }
 thiserror = "1.0"
+tracing = "0.1"
 url = "2.3"
 
 [dev-dependencies]
-tracing = "0.1"
```

Comment:
If not moved into another PR this change must at least be extracted in a specific commit fixing `cargo doc`.

Reply:
extracted into #2160